src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1079,7 +1074,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 
 				isNewWindowOperation = this.isNewWindowOperation(e);
 
-				const draggedEditors = [];
+				const draggedEditors = []; // TODO review
 				const selectedEditors = this.groupView.selectedEditors;
 				const isMultiSelected = this.groupView.isSelected(editor) && selectedEditors.length > 1;
 				if (isMultiSelected) {
```

Comment:
Can this be simplified? why would this not always be `selectedEditors`?

Reply:
You are right, this seems to be possible now